ballerina/tests/test.bal

```diff
@@ -0,0 +1,284 @@
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testDeleteRole() returns error? {
+    // Define the role ID to delete
+    int roleId = 10009; 
+    http:Response response = check jira->/rest/api/'3/role/[roleId].delete();
+
+    // Assertions to verify that deletion was successful
+    test:assertEquals(response.statusCode, 204, "Expected 204 No Content status for successful deletion.");
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testCreateProject() returns error? {
+
+    // Call the `createProject` endpoint.
+    ProjectIdentifiers response = check jira->/rest/api/'3/project.post(
+        payload=
+        {
+            key: "EX",
+            name: "Example",
+            projectTypeKey: "business",
+            leadAccountId:"712020:5d71be5f-debd-474f-9ec9-a97b9023ea4e"
+        }
+    );
+    test:assertNotEquals(response.id,null, "Unsuccessful project creation");
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetProjectSecurityLevelLive() returns error? {
+    string projectKeyOrId = "EX";  // Specify a valid project key or ID here
+
+    // Call the live API to retrieve the security level details for the project.
+    ProjectIssueSecurityLevels response = check jira->/rest/api/'3/project/[projectKeyOrId]/securitylevel();
+
+    // Assertions to verify expected properties in the live response
+    test:assertNotEquals(response.levels.length(), 0, "No security levels found for project.");
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetApplicationRoles() returns error? {
+    ApplicationRole[] response = check jira->/rest/api/'3/applicationrole();
+
+    // Validate the array is not empty and has expected properties in each item.
+    test:assertNotEquals(response.length(), 0, "Expected non-empty application roles array");
+
+    foreach var role in response {
+        test:assertNotEquals(role.name, "", "Application role name should not be empty");
+        test:assertNotEquals(role.key, "", "Application role key should not be empty");
+    }
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetSpecificApplicationRole() returns error? {
+    string ProjectKey = "jira-software";
+
+    ApplicationRole response = check jira->/rest/api/'3/applicationrole/[ProjectKey];
+
+    // Assertions
+    test:assertEquals(response.key, "jira-software", msg = "Expected key to be 'jira-software'");
+    test:assertEquals(response.name, "Jira Software", msg = "Expected name to be 'Jira Software'");
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetAuditRecords() returns error? {
+    // Call the function without any query parameters
+    AuditRecords response = check jira->/rest/api/'3/auditing/'record();
+    // Assertions
+   test:assertNotEquals(response.records, null, "Expected 'records' array to be non-empty");
```

Comment:
Ditto. Please make sure to assert the values. Because asserting specific values gives you stronger, more meaningful tests